src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,87 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  postId: number,
+  addComment: (
+    name: string,
+    email: string,
+    body: string,
+    postId: number
+  ) => Promise<void>
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  postId, addComment,
+}) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [value, setValue] = useState({
+    name: '',
+    email: '',
+    body: '',
+  });
```

Comment:
make outside component constant with initial values and use in both places.
same for isError state
```
const initialValue = {
  name: '',
  email: '',
  body: '',
}
```
```suggestion
  const [value, setValue] = useState(initialValue);
```